optuna/pruners/_hyperband.py

```diff
@@ -24,6 +24,10 @@ class HyperbandPruner(BasePruner):
     `Hyperband <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`__ attacks this trade-off
     by trying different :math:`n` values for a fixed budget.
 
+    The pruner handles NaN values in the following manner:
+        1. If the final objective  values are NaN, the trial will not be pruned.
```

Comment:
```suggestion
        1. If the final objective values are NaN, the trial will not be pruned.
```
There is an extra space in this line.